javascript/javascript1/week1/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>JavaScript1 week-1</title>
+</head>
+<body>
+    <div class="agecalculator">
```

Comment:
usually, we use the `kebab-case` convention for classes names - for example `age-calculator`

Reply:
I will keep this in mind.